connectors/email/src/main/java/io/camunda/connector/email/client/jakarta/inbound/PollingManager.java

```diff
@@ -138,7 +138,7 @@ private boolean correlate(Email email) {
             case CorrelationFailureHandlingStrategy.ForwardErrorToUpstream ignored -> {
               this.connectorContext.log(
                   Activity.level(Severity.ERROR)
-                      .tag("ForwardErrorToUpstream")
+                      .tag("Message") // TODO: standardize tags
```

Comment:
Is that TODO needed as `ActivityLogTag` was introduced?

Reply:
not needed anymore :)